packages/odata-common/src/properties-util.spec.ts

```diff
@@ -1,18 +1,14 @@
-import { TestEntity } from '@sap-cloud-sdk/test-services/v2/test-service';
+import { CommonEntity } from '../test/common-entity';
 import { isNavigationProperty } from './properties-util';
 
 describe('properties-util', () => {
   describe('isNavigationProperty', () => {
     it('returns true for one to one navigation property', () => {
-      expect(isNavigationProperty('toSingleLink', TestEntity)).toBe(true);
-    });
-
-    it('returns true for one to many navigation property', () => {
-      expect(isNavigationProperty('toMultiLink', TestEntity)).toBe(true);
+      expect(isNavigationProperty('toSingleLink', CommonEntity)).toBe(true);
     });
 
     it('returns false for boolean property', () => {
-      expect(isNavigationProperty('booleanProperty', TestEntity)).toBe(false);
+      expect(isNavigationProperty('stringProperty', CommonEntity)).toBe(false);
```

Comment:
Ah, here we have a negative test. I think one level of `describes` can still be removed here.

Reply:
Also, as we have moved the multilink test to v2, should there be another test for v4 as well?

Reply:
added it to this ticket: https://github.com/SAP/cloud-sdk-backlog/issues/463 about test parity.